use-cases/credit_risk/risk_bucketing.ipynb

```diff
@@ -0,0 +1,932 @@
+{
```

Comment:
<p>Please clear all notebook outputs. Feel free to include an extra markdown table or PNG file if you want to show any resulting tables or graphs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
That was done for last commit